pom.xml

```diff
@@ -30,7 +30,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
     <jenkins.baseline>2.479</jenkins.baseline>
-    <jenkins.version>${jenkins.baseline}.1</jenkins.version>
+    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
```

Comment:
why?  there is no need to do this in an API plugin. This causes anyone consuming this via a bom update to have to also bump their versions.
https://github.com/jenkinsci/jackson2-api-plugin/pull/307#issuecomment-3097318840